.travis.yml

```diff
@@ -20,11 +19,16 @@ services:
 - mongodb
 - redis-server
 - cassandra
+- mysql
+- memcached
 before_install:
-- phpenv config-rm xdebug.ini || return 0
+  - composer install --prefer-source --no-interaction
+  - vendor/bin/install-phalcon.sh 2.0.x
+  - phpenv config-rm xdebug.ini || return 0
 install:
 - composer install --no-interaction
```

Comment:
why is `composer install` also in `install`? How about you move the phalcon installation to `install` instead?